.github/CHANGELOG.md

```diff
@@ -49,15 +49,34 @@
   For example, if we wanted to only push single-qubit gates through
   controlled gates and cancel adjacent inverses, we could do:
 
+  ```python
+  from pennylane.transforms import commute_controlled, cancel_inverses
+  pipeline = [commute_controlled, cancel_inverses]
+
+  @qml.qnode(dev)
+  @qml.compile(pipeline=pipeline)
+  def qfunc(x, y, z):
```

Comment:
The previous `qfunc` definition was decorated with `@qml.compile`, hence we need to redefine `qfunc`.